packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.tsx

```diff
@@ -37,8 +37,6 @@ const BuySellModal = () => {
             if (general_store.is_form_modified) {
                 showModal({
                     key: 'CancelAddPaymentMethodModal',
-                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                    // @ts-ignore TODO: fix typings in CancelAddPaymentMethodModal and make them optional and remove this comment
                     props: {},
```

Comment:
do we need to give the props field here?

Reply:
yeah, @adrienne-deriv had informed me that we have to pass props: {} here even though we're not passing anything to the modal, if the modal has props that can be passed through, we must pass empty props here. And the error for @ts-ignore was because we didn't fix the typings for that modal prior

![Screenshot 2023-09-08 at 4 46 03 PM](https://github.com/farrah-deriv/deriv-app/assets/103412909/9efe5292-d1ce-4e3e-bc87-a6f0805664db)

Reply:
oh.. then i think for many existing refactored cards we will have to add props field since we removed it as part of refactoring.

Reply:
hmmm could be but i think majority of the modals that have props, we do pass some props to them so it should be ok